data/registers/timer_v1.yaml

```diff
@@ -755,15 +753,6 @@ enum/ETF:
   - name: FDTS_Div32_N8
     description: fSAMPLING=fDTS/32, N=8
     value: 15
-enum/ETP:
```

Comment:
i'd keep this one, "polarity = false" is not very clear. If the bit was named "invert" then removing it would make more sense.